app/build.gradle

```diff
@@ -1,16 +1,16 @@
 plugins {
     id 'com.android.application'
     id 'org.jetbrains.kotlin.android'
-    id 'com.google.devtools.ksp' version "1.6.10-1.0.4"
+    id 'com.google.devtools.ksp' version "1.7.21-1.0.8"
```

Comment:
Can we go to the latest stable version of both Kotlin and KSP? For Kotlin, it's `1.8.10` and for KSP, it's `1.8.10-1.0.9`